MergerService/src/run.cs

```diff
@@ -222,8 +220,7 @@ private void RunTask(MergeTask task, ITaskUtils taskUtils)
                                             try
                                             {
                                                 task.Percentage = 100 * (tileProgressCount / totalTileCount);
-                                                UpdateParameters updateParameters = task.GetUpdateParameters();
-                                                taskUtils.UpdateTask(task.JobId, task.Id, updateParameters);
+                                                taskUtils.UpdateProgress(task.JobId, task.Id, task.Percentage);
```

Comment:
progress log should probably include job and task id and be debug level

Reply:
Added and changed.